### PR TITLE
a few notes on packrat

### PR DESCRIPTION
**Pull request by benmarwick**
Following up on https://github.com/travis-ci/travis-ci/issues/6504 with @jimhester & @craigcitro

### CODE REVIEW

File: user/languages/r.md [hunk 1]
```diff
@@ -188,6 +188,25 @@ r: bioc-release
...
+```yaml
+install:
+  - R -e "0" --args --bootstrap-packrat
+```
+
+You can minimise build times by caching your packrat packages with:
+
+```yaml
+cache:
+  directories: /home/travis/build/<username>/<repository>/packrat/
```
Comment (jimhester) [inline user/languages/r.md]: This should probably be `$TRAVIS_BUILD_DIR/packrat`

File: user/languages/r.md [hunk 2]
```diff
@@ -188,6 +188,25 @@ r: bioc-release
 Travis CI will use the proper R version for that version of Bioconductor and
 configure Bioconductor appropriately for installing dependencies.
 
+### Packrat
+
+If you want Travis CI to use your project-specific [packrat](https://rstudio.github.io/packrat/) package library, rather than the default behaviour of downloading your package dependencies from CRAN, you can add this to your `.travis.yml`:
+
+```yaml
+install:
+  - R -e "0" --args --bootstrap-packrat
```
Comment (jimhester) [inline user/languages/r.md]: I believe the `-e "0"` is superfluous and can be removed.

File: user/languages/r.md (hunk 1 above)
Comment (benmarwick) [inline user/languages/r.md]: Right, thanks, good idea, I wasn't aware of that, but it works fine on my example: https://travis-ci.org/benmarwick/mjbtramp/builds/158877245

File: user/languages/r.md (hunk 2 above)
Comment (benmarwick) [inline user/languages/r.md]: I haven't had much luck with this, when I remove `-e "0"` the build stalls and quits: https://travis-ci.org/benmarwick/mjbtramp/builds/158881647, but works when I put it back in: https://travis-ci.org/benmarwick/mjbtramp/builds/158882619

**Comment by benmarwick** [comment]
Is there anything else I should add to improve this PR?

File: user/languages/r.md (hunk 2 above)
Comment (jimhester) [inline user/languages/r.md]: Ok LGTM then.

**Pull request merged**